src/utils/externalCbnftUrl.ts

```diff
@@ -0,0 +1,12 @@
+import coke from '@/config/partners/coke'
+
+export function externalCbnftUrl({ address }: { address: string }) {
+  const allDrops = [...coke.drops]
```

Comment:
you could do

```suggestion
import { partners } from '@/config/partners'
import { Drop } from '@/config/partners/types'

export function externalCbnftUrl({ address }: { address: string }) {
  const allDrops = partners.reduce((acc, partner) => {
    return [...acc, ...partner.drops]
  }, [] as Drop[])
```

But what is this PR accomplishing?

Reply:
Thanks for the intel. The intention of this PR is to make it so that mints that we support (ThirdWeb contracts, for example) open on CBNFT.

Reply:
Gotcha - this should work then

Reply:
@gio-m-cb do you support native minting for all of these on your page? can you link an example please?

Reply:
Yep, native minting is supported for TW contracts, except 1155